lib/fluent/plugin/in_mongo_tail.rb

```diff
@@ -133,9 +133,9 @@ def get_collection
     def node_string
       case
       when @database
-        "#{@host}:#{@port}"
+        ["#{@host}:#{@port}"]
       when @url
-        @url
+        "#{@url}"
```

Comment:
[nits] it seems not to be needed since `@url` is already a string.  https://github.com/fluent/fluent-plugin-mongo/pull/122/files#diff-4180e7be83581f1c09f58b1cb9d2b41aR29

Reply:
Agreed @ganmacs 
Should I send pull request again?

Reply:
Please commit the change in this PR 🙏

Reply:
Commit done & available for your review. @ganmacs

Reply:
cb8d4ef and 6858052 have the same commit message. Please make the commit message more descriptive (it looks nits, but I think it's important for other developers).

Reply:
Added proper commit message in [023e192](https://github.com/fluent/fluent-plugin-mongo/pull/122/commits/023e192fce879d208e0a296f5dfd51d0ec8cff3f) please review it.